Google.PowerShell.IntegrationTests/Compute/Compute.GceInstance.Tests.ps1

```diff
@@ -160,6 +160,21 @@ Describe "New-GceInstanceConfig" {
         $instanceConfig.NetworkInterfaces.NetworkIP | Should BeExactly $address
     }
 
+    It "should accept -CustomMemory and -CustomCpu" {
+        $instanceConfig = New-GceInstanceConfig -Name $instance -Disk $attachedDisk `
+                                                -CustomCpu 2 -CustomMemory 2048
+        $instanceConfig.MachineType | Should Be "custom-2-2048"
+    }
+
+    It "should error out for -CustomMemory and -CustomCpu" {
+        { New-GceInstanceConfig -Name $instance -Disk $attachedDisk `
+                              -MachineType "f1-micro" `
+                              -CustomCpu 2 -CustomMemory 2048 } | Should Throw "cannot be used with -MachineType"
+
+        { New-GceInstanceConfig -Name $instance -Disk $attachedDisk `
+                              -CustomCpu 2 } | Should Throw "-CustomCpu must be used together"
+    }
+
```

Comment:
Do we want to start writing unit tests, now that I figured out how? They run much faster than these functional tests, and don't need the authentication secret (so we could run them on pull requests).